deepview_profile/analysis/request_manager.py

```diff
@@ -112,9 +108,24 @@ def _handle_analysis_request(self, analysis_request, context):
                 context,
             )
 
+            # send ddp analysis
+            if self._early_disconnection_error(context):
+                return
+            
+            if analysis_request.ddp_analysis_request:
+                ddp_resp = next(analyzer)
+                self._enqueue_response(
+                    self._send_ddp_response,
+                    ddp_resp,
+                    context,
+                )
+            
+            next(analyzer)
+
+            # execution time
             elapsed_time = time.perf_counter() - start_time
             logger.debug(
-                'Processed analysis request %d from (%s:%d) in %.4f seconds.',
+                "Processed analysis request %d from (%s:%d) in %.4f seconds.",
```

Comment:
Why are these (and many like it) changed to single quotes?

Reply:
From blacked formatters, maybe we can make a default formatter guide